cmd/manager/main.go

```diff
@@ -58,6 +58,12 @@ func printVersion() {
 }
 
 func main() {
+	if err := start(); err != nil {
```

Comment:
Why split this? Don't we error during the start process anyway?

Reply:
Good practices in go to leave main clean. Just code readability.

Reply:
My only issue with this is that we don't do it in our other operators and its different to what is templated from the SDK. If you feel strongly about it I'm happy to leave it otherwise I'd prefer it to match our other operators for consistency.

Reply:
I dont feel strong, but its just good practice :) and would be nice to move other too